src/ring_theory/ideal/operations.lean

```diff
@@ -1174,7 +1212,54 @@ surjective_quot_mk _
 lemma quotient.mkₐ_ker (I : ideal A) : (quotient.mkₐ R I).to_ring_hom.ker = I :=
 ideal.mk_ker
 
-variable {R}
+variables {R} {B : Type*} [comm_ring B] [algebra R B]
+
+lemma quotient.smul (f : A →ₐ[R] B) (r : R) (x : f.to_ring_hom.ker.quotient) :
```

Comment:
I think this should be called `ker_lift.map_smul`

Reply:
Either that, or you should just prove it within the definition of `ker_lift_alg`